CompDMD_Location.m

```diff
@@ -38,8 +39,9 @@
 
 %% output
 parameters = getMultiStarts(parameters, objectiveFunction, optionsPesto);
-optx = reshape(parameters.MS.par(:,1),[N,L]);
-[SPToolboxFolder,~,~]=fileparts(which('CompDMD_Location'))
-filename=sprintf('%s%i%s%i%s','DMDinfo\B_SP_dim',N,'points',L,'.csv');
+optx = reshape(parameters.MS.par(:,1),[N,L-1]);
+optx = [optx,-sum(optx,2)];
+[SPToolboxFolder,~,~]=fileparts(which('CompDMD_Location'));
+filename=sprintf('%s%i%s%i%s','DMDTrueMeanInfo\B_SP_dim',N,'points',L,'.csv');
 dlmwrite(fullfile(SPToolboxFolder,filename),optx,'delimiter',',','precision',12);
```

Comment:
There was already a little error in the old version. Using the "\", you have to watch out, since this is only correct for windows. On LInux and macOS, you have to use "/". So it would be nice to have a little "if" here, that checks this and sets the paths accordingly...